python/lsst/pipe/tasks/imageDifference.py

```diff
@@ -87,6 +87,10 @@ class ImageDifferenceConfig(pexConfig.Config):
         dtype=bool,
         default=True
     )
+    astrometer = pexConfig.ConfigurableField(
+        target = measAstrom.AstrometryTask,
+        doc = "astrometry task; used to match sources to reference objects, but not to fit a WCS",
+    )
```

Comment:
It looks like you're touching all the places where the astrometry subtask is used here; could we rename it at the same time to `astrometry` for consistency?

There are also other places we use `astrometer`, which I'd also like to change; I prefer just calling the subtask `astrometry`, since we tend to name other tasks after the "act of doing something" (e.g. `measurement`) not "a thing that does something" (`measurer`).